controllers/users.js

```diff
@@ -167,18 +167,45 @@ const getUsers = async (req, res) => {
     if (transformedQuery?.filterBy === OVERDUE_TASKS) {
       try {
         const tasksData = await getOverdueTasks(days);
```

Comment:
tasksData array to only include tasks that are assigned to the current user. However, the problem is that the tasksData array may contain tasks that are assigned to multiple users. This means that the userTasks array may contain duplicate tasks.